### PR TITLE
aarch64: improved log format and clean ups

### PR DESCRIPTION
**Pull request by chl337**
## Includes

* Pretty format for _struct PtRegs_
* Update outdated comment
* Distinguish output log from the kernel
* ~remove dead code~

### CODE REVIEW

File: qlib/kernel/SignalDef.rs [hunk 1]
```diff
@@ -112,13 +112,50 @@ pub struct PtRegs {
...
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "SP:[{:#x}]
+        write!(f, "\nSP:[{:#x}]
                    PC:[{:#x}]
                    State:[{:#x}]
                    X0:[{:#x}]
-                   GenRegs:[{:#?}]",
-                   self.sp, self.pc, self.pstate,
-                   self.orig_x0, self.regs)
+                   GenRegs:[
```
Comment (shrik3) [inline qlib/kernel/SignalDef.rs:119]: just out of curiosity,  can we apply something like `derive(debug)` to shorten this?

Comment (chl337) [reply qlib/kernel/SignalDef.rs:119]: I don't know. I have it like this because I don't want to count by myself.

Comment (shrik3) [reply qlib/kernel/SignalDef.rs:119]: doesn't "\n" work? why all the newlines?

File: qlib/kernel/SignalDef.rs [hunk 2]
```diff
@@ -856,7 +893,8 @@ pub struct SignalStruct {
     trapStopPending: bool,
 }
 
-// https://elixir.bootlin.com/linux/latest/source/arch/x86/include/uapi/asm/signal.h#L132
+// Based on linux's 'struct signalstack' in signal.h
```
Comment (shrik3) [inline qlib/kernel/SignalDef.rs:896]: in this case please update the link :D  If they are the same, keep at least one.

File: qlib/kernel/arch/x86_64/signal.rs [hunk 3]
```diff
@@ -1,122 +0,0 @@
-// Copyright (c) 2021 Quark Container Authors / 2018 The gVisor Authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (shrik3) [inline qlib/kernel/arch/x86_64/signal.rs:3]: This should go to the main, not aarch64 feature branch

Comment (chl337) [reply qlib/kernel/arch/x86_64/signal.rs]: Restored deleted file.

File: qlib/kernel/SignalDef.rs (hunk 2 above)
Comment (chl337) [reply qlib/kernel/SignalDef.rs:865]: Added a reference.

File: qlib/kernel/SignalDef.rs (hunk 1 above)
Comment (chl337) [reply qlib/kernel/SignalDef.rs]: Adjusted.

**Review by shrik3 — APPROVED** [review]

**Pull request merged**